modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_file_commands.adoc

```diff
@@ -120,6 +122,32 @@ For example, if the user's id was "john" and the `--path` value was "tree/apple"
 
 `files:cleanup` tidies up the server's file cache by deleting all file entries that have no matching entries in the storage table.
 
+== The files:remove-storage command
+
+When a mount point gets removed, the storage ID and file cache related info stay in the database. This can over time consume considerable amount of database space respectively can slow down database queries. Withe the `files:remove-storage` command, these orphans can be removed. As a rule of thumb though not necessary in any case, using this command should be considered to run in single user mode to avoid any conflicting access.
```

Comment:
```suggestion
When a mount point gets removed, the storage ID and file cache related info stay in the database. Over time, this can consume a considerable amount of database space and can slow down database queries. With the `files:remove-storage` command, these orphans can be removed. As a rule of thumb, though not essential, this command should be run in single user mode to avoid any conflicting access.
```